CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -908,6 +909,14 @@ - (void)showBanner:(CPAppBanner*)banner force:(BOOL)force {
                     pasteboard.string = voucherCode;
                 }
             }
+
```

Comment:
Please request location permission only without including the CleverPushLocation SDK here. We could tell the customer to include it manually and then we can use it via Reflection - see https://stackoverflow.com/questions/7550012/does-objective-c-have-reflection